docker/docker-compose.yml

```diff
@@ -5,18 +5,19 @@ services:
       context: ..
       dockerfile: docker/Dockerfile
       args:
-        - TAG=humble-0.21
+        - TAG=foxy-0.21
     environment:
       DISPLAY: $DISPLAY
       XAUTHORITY: $XAUTHORITY
       NVIDIA_DRIVER_CAPABILITIES: all
       ROS_LOG_DIR: /tmp
     container_name: snp
-    image: ghcr.io/ros-industrial-consortium/scan_n_plan_workshop:humble-latest
+    image: ghcr.io/ros-industrial-consortium/scan_n_plan_workshop:foxy-latest
     stdin_open: true
     tty: true
     network_mode: host
-    user: ${CURRENT_UID}  # CURRENT_UID=$(id -u):$(id -g)
+    privileged: true
+    user: root #${CURRENT_UID}  # CURRENT_UID=$(id -u):$(id -g)
```

Comment:
Let's revert this change. I think you have to be a non-root user to be able to access the graphics drivers to visualize Rviz and other graphical programs